packages/contracts/contracts/discovery/Project.sol

```diff
@@ -15,6 +15,16 @@ contract Project is IProject, ERC165 {
     // must be deployed via the Controller
     // will have a similar role as the CTND Vesting contract
 
+    //
+    // Constants
+    //
+
+    uint256 public constant MUL = 10**18;
```

Comment:
will this need to change, in light of the recent changes in the ctnd sale contract?

Reply:
I think so. The investment token is aUSD, so we still have that issue.

Reply:
correct